colors.css

```diff
@@ -0,0 +1,19 @@
+/* Light theme */
```

Comment:
NIT: this file should probably have a name that reflects the fact it is a template...  When I scan code/repos and I see a css file, I don't expect that to be a template